mavproxy/source/docs/modules/ublox.rst

```diff
@@ -1,5 +1,5 @@
 ==========================
-uBlox GPS Managment Module
+uBlox GPS Management Module

```

Comment:
need to fix overline and underline length

Reply:
Fixed, Thanks.